google-cloud-core/src/main/java/com/google/cloud/Policy.java

```diff
@@ -240,7 +326,7 @@ public final Builder setEtag(String etag) {
      * Sets the version of the policy. The default version is 0, meaning only the "owner", "editor",
      * and "viewer" roles are permitted. If the version is 1, you may also use other roles.
      */
-    protected final Builder setVersion(int version) {
+    public final Builder setVersion(int version) {
```

Comment:
What happens if a user sets an invalid version here?

Reply:
For now it's kept flexible. The API will error with a corresponding error. Sample code will also address setting this field specifically.